api/s2n.h

```diff
@@ -110,6 +110,15 @@ extern int s2n_config_set_extension_data(struct s2n_config *config, s2n_tls_exte
 extern int s2n_config_send_max_fragment_length(struct s2n_config *config, s2n_max_frag_len mfl_code);
 extern int s2n_config_accept_max_fragment_length(struct s2n_config *config);
 
+extern int s2n_config_set_session_state_lifetime(struct s2n_config *config, uint64_t lifetime_in_nanos);
```

Comment:
nanos seems like awfully high precision for a lifetime.  any reason not to make this seconds based?

Reply:
Currently, the lifetime is in nanoseconds. Not sure as to why it was set like that. Maybe because the monotonic_clock is in nanoseconds and we don't want to convert it to seconds when we serialize and deserialize resumption state.

(defined in s2n_resume.h)
#define S2N_STATE_LIFETIME_IN_NANOS     21600000000

Reply:
you can store it in nanoseconds, but have the API take it in seconds and do the conversion at set time.